src/rules/store/avoid-combining-selectors.ts

```diff
@@ -40,14 +40,7 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
         node: TSESTree.CallExpression,
```

Comment:
I think you could achieve the same result using the existent selectors, no?

```suggestion
    const pipeableOrStoreSelect = `:matches(${storeExpression(
      storeName,
    )}[callee.property.name='pipe']:has(CallExpression[callee.name='select']), ${storeSelect(
      storeName,
    )})`

    return {
      [`CallExpression[callee.name='combineLatest'][arguments.length>1] ${pipeableOrStoreSelect} ~ ${pipeableOrStoreSelect}`](
        node: TSESTree.CallExpression,
      ) {
        context.report({
          node,
          messageId,
        })
      },
```

Reply:
Nice! 🤩